const pi = Math.PI;

try {
    pi = 3;

} catch (error) {
    console.log(error);
}
